sdk/communication/Azure.Communication.PhoneNumbers/src/PhoneNumbersClient.cs

```diff
@@ -197,12 +197,12 @@ public virtual UpdatePhoneNumberCapabilitiesOperation StartUpdateCapabilities(st
             }
         }
 
-        /// <summary> Gets the details of the given acquired phone number. </summary>
-        /// <param name="phoneNumber"> The acquired phone number whose details are to be fetched in E.164 format, e.g. +14255550123. </param>
+        /// <summary> Gets the details of the given purchased phone number. </summary>
+        /// <param name="phoneNumber"> The acquired phone number whose details are to be fetched in E.164 format, e.g. +11234567890. </param>
```

Comment:
Lets change the e.g. to `14255550123` this is an actual invalid number we are using across langs

Reply:
Fixed